e2e/run-test.sh

```diff
@@ -174,10 +179,17 @@ run_test "Build should push to a private registry with TLS enabled" \
           e2e/builds/tls_with_basic_auth.yaml \
           test-tls-with-basic-auth \
           "$namespace"
+
 run_test "Build should pull base image from a private registry" \
           e2e/builds/private_base_image.yaml \
           test-private-base-image \
           "$namespace"
-verify_image
+verify_image variable-base-app /app $BASE_DIR/internal/archive/testdata/expected/simple-app
```

Comment:
I would move `interna/archive/testdata/{simple-app,init-container}` out of that directory and  nested it under `./e2e`. The former indicates it's test data for the "archive" tests which it is not.

Reply:
done